generate-renode-scripts.py

```diff
@@ -151,9 +163,16 @@ def generate_cpu():
     kind = constants['config_cpu_type']['value']
 
     if kind == 'VEXRISCV':
```

Comment:
Privilege support is only correct on the Linux configuration of vexriscv.

Reply:
Now privilege architecture and `A` instruction set support is enabled only for cpu `LINUX` variant.